client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -135,7 +136,9 @@ const DomainPicker: FunctionComponent< Props > = ( {
 		}
 	}, [ allSuggestions, currentDomain ] );
 
-	useTrackModal( 'DomainPicker' );
+	useTrackModal( 'DomainPicker', () => ( {
+		selected_domain: getSelectedDomain()?.domain_name,
```

Comment:
This records the selected domain even if the picker was closed with the cross (which I think of as cancel). It makes no distinction between whether the selection was confirmed or not.

Not necessarily wrong from an analytics points of view, just interesting.

The `calypso_traintracks_interact` action does the same so 🤷

Reply:
hmm that is interesting, yea they probably should be only recording it if you've actually confirmed your selection...

Reply:
`selected_domain === ""` means cancelled, `selected_domain === "something"` means confirmed?

Reply:
hmm actually when I change the selection and then close the modal with the cross, it is recording the old (currently selected) domain as I'd expect.